### DISCUSSION TITLE
VRoid SDK for Unreal Engine 0.0.1にて、読み込んだVRMによってはアニメーションが適用されない

### DISCUSSION THREAD

**Discussion by SigRem221** [pixiv/vroid-sdk-developers#121] [Q&A] [answered]
VRoid SDK for Unreal Engine 0.0.1を早速インストールし、「D&DでVRMモデルを読み込む機能」を試してみたのですが、
公式サンプルのVRMモデルでは正常に読み込みが完了してアニメーションもリターゲットが機能するのに対し、
Unityを経由してfbxモデルから作成したVRMモデルではTポーズのままリターゲットが機能しない現象が起きてしまいました。

UniVRM経由では正しく動かないものなのでしょうか？
また、UniVRMを使用せずfbxモデルからUEまでVRMをインポートできる手段などはありますでしょうか？

**Comment by ytomi-pixiv** [comment, marked as answer]
質問ありがとうございます！
VRoid SDK for UEにおけるリターゲティングについてですが、preview版ではVRoid Studioから出力したモデルを中心に動作確認を行っています。UniVRMから出力したモデル一般については、考慮すべきケースが多く、まだカバーできていないものが多いです。そのため、連携時のモデル一覧では、Warningアイコンを付与しています（[ドキュメント](https://vroid.notion.site/VRoid-SDK-for-Unreal-Engine-21321b5e31b247e99600f270b3414d4e#dfec4e56023f42f38ea8e5856b5d2401:~:text=%E3%83%A2%E3%83%87%E3%83%AB%E3%81%AE%E3%82%B5%E3%83%A0%E3%83%8D%E3%82%A4%E3%83%AB%E5%8F%B3%E4%B8%8A%E3%81%ABWarning%E3%82%A2%E3%82%A4%E3%82%B3%E3%83%B3%E3%81%8C%E3%81%A4%E3%81%84%E3%81%A6%E3%81%84%E3%82%8B%E3%83%A2%E3%83%87%E3%83%AB%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6)）。

正式版に向け、対応範囲を増やしていきたいと思っております。
お手数おかけしますが、もしよろしければ問題の発生したモデルデータを問い合わせ窓口からお送りいただけますでしょうか。
本スレッドのURLとともにお送りいただけますと幸いです。
https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk
